features/step_definitions/skin_steps.rb

```diff
@@ -173,7 +178,7 @@
 Then(/^the cache of the skin on "(.*?)" should expire after I save a parent skin$/) do |arg1|
   skin = Skin.find_by_title(arg1)
   orig_skin_key = skin_cache_value(skin)
-  parent_id = SkinParent.where(:child_skin_id => skin.id).last.parent_skin_id
+  parent_id = SkinParent.where(child_skin: skin.id).last.parent_skin_id
```

Comment:
I'm pretty sure the test is failing here because this needs to be child_skin_id: instead of child_skin: